### ISSUE TITLE
Is this bundle still maintened ?

### ISSUE THREAD

**Issue by erichard** [antimattr/GoogleBundle#81]
This bundle is quite useful but I wonder if it is still maintened.

No commit since April 2015. Multiples PR waiting for a merge. Multi issues without answers.
And no Symfony 3 support...

@matthewfitz Are you still around ? Do you need help ?

**Comment by crash21** [comment]
Indeed, this is an interesting bundle but seems abandoned. Any alternatives?

**Comment by EricReiche** [comment]
Hi,

we ( @flix-tech ) forked this bundle in 2012 after #17 wen't unanswered.
https://github.com/meinfernbus/GoogleBundle

There are obviously some differences in functionality since then, so we treat as independent, but it is maintained.

**Comment by sethrea** [comment]
@EricReiche at first glance, your Analytics / eCommerce seems slightly out dated, for example Impressions are missing, do you plan to address these in near future?

I'm writing here since Issues don't seem accessible for your repo.

**Comment by EricReiche** [comment]
@sethrea good point, I enabled issues.

As I said, we split in 2012 and impression were added on this version in 2015. Currently there is no plan to integrate impressions, but feel free to open a pull request.

**Comment by ernestgm** [comment]
Google has updated some parameters of analytis such as the registration of the currency with which the transaction is performed. They could update the bundle to allow that functionality. Personally I had to modify the local bundle to be able to do it, but then I can not update the bundle.

**Comment by EricReiche** [comment]
@ernestgm would you mind sharing the changes on your account then?

**Comment by greg0ire** [comment]
I shot @matthewfitz an email, let's see what happens.
